app-text/ghostscript_gpl/ghostscript_gpl-9.25.recipe

```diff
@@ -0,0 +1,131 @@
+SUMMARY="An interpreter for the PostScript language and for PDF"
+DESCRIPTION="Ghostscript is a suite of software based on an interpreter for \
+Adobe Systems' PostScript and Portable Document Format (PDF) page description \
+languages. Its main purposes are the rasterization or rendering of such page \
+description language files, for the display or printing of document pages, and \
+the conversion between PostScript and PDF files."
+HOMEPAGE="https://ghostscript.com/"
+COPYRIGHT="2001-2018 Artifex Software, Inc."
+LICENSE="AGPL-3.0"
+REVISION="1"
+SOURCE_URI="https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs925/ghostscript-$portVersion.tar.gz"
+CHECKSUM_SHA256="baafa64740b090bff50b220a6df3be95c46069b7e30f4b4effed28316e5b2389"
+SOURCE_DIR="ghostscript-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
There is no reason to have a secondary arch package since the resulting binaries would do the same thing anyway. We only need them because this package won't build on `x86_gcc2` primary arch.

```diff
+if [ "$targetArchitecture" == "x86_gcc2" ]; then
-SECONDARY_ARCHITECTURES="x86"
+	SECONDARY_ARCHITECTURES="x86"
+fi
```

Reply:
What do you think of this variant?
```bash
if [ "$targetArchitecture" = x86_gcc2 ]; then
SECONDARY_ARCHITECTURES="x86"
fi
```